pkg/releaseindex/spec.go

```diff
@@ -0,0 +1,40 @@
+package releaseindex
+
+type Component string
```

Comment:
LMK if this makes sense to rename `Component` to `Authority`.

Reply:
I'm more keen to rename the thing here than changing `versionbundle` repo. I feel this would have long tail. It was called component in e2e-harness. I'm fine with renaming this to `Authority` which is at least consistent. But I don't want to touch `versionbundle` ATM.

Reply:
Renamed to `Authority`.